msosyak/sonic-buildimage
# Keep containers hostname up to date

**- What I did**
Implemented logic to keep containers hostname up to date

**- How I did it**
* Added updateHostName function to docker_image_ctl.j2
* Added hostname specification on container creating step
* Added listener for hostname changes in hostcfgd which call updateHostName for all running containers if an appropriate script(generated from docker_image_ctl.j2) exist in /usr/bin 

**- How to verify it**
Change hostname in config_db.json
Load new configuration
Check hostname in containers

**- Description for the changelog**
Added logic to keep containers hostname up to date


@akokhan @pyadvichuk @vsenchyshyn
